bcda/bcdacli/cli.go

```diff
@@ -361,8 +361,8 @@ func setUpApp() *cli.App {
 		},
 		{
 			Name:     "import-cclf-package",
```

Comment:
Can you explain the reasoning behind using this new CLI instead of `import-cclf-directory`?  I think it would be most excellent if we were actively executing code internally/testing that will be used in real world.

Reply:
import-cclf-directory is an existing function that imports files that are already in a directory.  Under the covers this code calls that function, but it needs to do some external prep work as well, hence the different name.

Reply:
Got it!  To avoid confusion, it might make sense to do a few things to ensure we aren't using this by accident in `prod`.
Can we rename to `import-synthetic-cclf-package`?  And maybe we could also even protect it so that its only usage in `DEBUG` mode?

Reply:
I can rename it.  I expect that it will in fact run in production so we can continue to test against production with data we can actually access.

Reply:
Yeah, hoping we can make that happen.  The rename to include the term `synthetic` should be sufficient.  Thanks!

Reply:
Rename is done in the last commit.